From de62967c73e27d133a9ffb69d6704e1f154500d8 Mon Sep 17 00:00:00 2001
From: emi <emiliorizzo@gmail.com>
Date: Sat, 26 Sep 2020 20:25:04 -0300
Subject: [PATCH] Add BigTreeTech platformio.ini

---
 platformio.ini | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/platformio.ini b/platformio.ini
index f8ec029de3..12c2ca8890 100644
--- a/platformio.ini
+++ b/platformio.ini
@@ -18,7 +18,7 @@
 [platformio]
 src_dir      = Marlin
 boards_dir   = buildroot/share/PlatformIO/boards
-default_envs = mega2560
+default_envs = STM32F103RC_btt_512K
 include_dir  = Marlin
 
 #
@@ -205,7 +205,7 @@ HAS_TMC26X              = TMC26XStepper=https://github.com/trinamic/TMC26XSteppe
                           src_filter=+<src/module/TMC26X.cpp>
 HAS_L64XX               = Arduino-L6470@0.8.0
                           src_filter=+<src/libs/L64XX> +<src/module/stepper/L64xx.cpp> +<src/gcode/feature/L6470>
-NEOPIXEL_LED            = Adafruit NeoPixel@1.5.0
+NEOPIXEL_LED            = https://github.com/bigtreetech/Adafruit_NeoPixel
                           src_filter=+<src/feature/leds/neopixel.cpp>
 MAX6675_IS_MAX31865     = Adafruit MAX31865 library@~1.1.0
 HAS_GRAPHICAL_LCD       = U8glib-HAL@~0.4.1
@@ -766,6 +766,7 @@ platform          = ${common_stm32f1.platform}
 extends           = env:STM32F103RC_btt
 board_upload.maximum_size=524288
 build_flags       = ${env:STM32F103RC_btt.build_flags} -DSTM32_FLASH_SIZE=512
+;custom_marlin.NEOPIXEL_LED = Adafruit NeoPixel=https://github.com/bigtreetech/Adafruit_NeoPixel
 
 [env:STM32F103RC_btt_512K_USB]
 platform          = ${common_stm32f1.platform}
-- 
2.25.1

